ads/_config.js

```diff
@@ -1233,6 +1233,11 @@ const adConfig = jsonConfiguration({
     renderStartImplemented: true,
   },
 
+  'verizonmedia': {
+    prefetch: 'https://jac.yahoosandbox.com/amp/jac.js',
+    preconnect: ['https://jill.fc.yahoo.com'],
```

Comment:
I see you are not using the renderStart API, have you looked into it? It should give you some performance gains but is optional.

Reply:
great thanks.  will add it and include in nexxt push